assets/training/finetune_acft_image/environments/acft_image_huggingface/context/Dockerfile

```diff
@@ -9,8 +9,6 @@ COPY requirements.txt .
 RUN pip install -r requirements.txt --no-cache-dir
 
 # Vulnerability fixes
-RUN pip install pydash==6.0.0
-RUN pip install urllib3==2.0.7
-RUN pip install pyarrow==14.0.1
-RUN pip install aiohttp==3.9.1
+RUN pip install 'cryptography>=42.0.2'
```

Comment:
>=?

Reply:
Thanks for catching this. Fixed to strict pin